src/core/thread/csl_tx_scheduler.hpp

```diff
@@ -105,10 +105,53 @@ class CslTxScheduler : public InstanceLocator, private NonCopyable
         bool      mCslSynchronized : 1; ///< Indicates whether or not the child is CSL synchronized.
         uint8_t   mCslChannel;          ///< The channel the device will listen on.
         uint32_t  mCslTimeout;          ///< The sync timeout, in seconds.
-        uint16_t  mCslPeriod;           ///< CSL sampled listening period in units of 10 symbols (160 microseconds).
-        uint16_t  mCslPhase;            ///< The time when the next CSL sample will start.
-        TimeMilli mCslLastHeard;        ///< Time when last frame containing CSL IE was heard.
-        uint64_t  mLastRxTimestamp;     ///< Time when last frame containing CSL IE was received, in microseconds.
+        uint16_t  mCslPeriod;           ///< CSL sampled listening period between channel samples in units of 10 symbols (160 microseconds).
```

Comment:
Maybe change like this?
```suggestion
        uint16_t  mCslPeriod;           ///< CSL sampled listening period between the beginning point of 2 consecutive channel samples in units of 10 symbols (160 microseconds).
```

Reply:
@Irving-cl Thanks for your review! Inserted "consecutive" but left out the "beginning point" part as a period can by definition be measured between any corresponding points, not just the beginning of the samples, do you agree?